database/seedGenerator.test.js

```diff
@@ -0,0 +1,55 @@
+const generateAllSeedData = require('./seedingUtils.js');
+
+describe('Seed Generation', () => {
+  describe('seed generation function', () => {
+    test('Creates one set of User and Project data', () => {
+      const single = generateAllSeedData(1);
+      expect(single.users.length).toBe(1);
+      expect(single.projects.length).toBe(1);
+    });
+
+    test('Creates multiple updates', () => {
+      const single = generateAllSeedData(1);
+      expect(single.updates.length).toBeGreaterThan(0);
```

Comment:
This will not necessarily check for multiple updates. A single update will pass this test.

Reply:
That's true. I was only really testing to see that one existed at all.